catalog-model/src/main/java/org/openecomp/sdc/be/model/operations/impl/GraphLockOperation.java

```diff
@@ -45,14 +45,7 @@
      */
     @Override
     public StorageOperationStatus lockComponent(String componentId, NodeTypeEnum nodeType) {
-        log.info("lock resource with id {}", componentId);
-        JanusGraphOperationStatus lockElementStatus = null;
-        try {
-            lockElementStatus = disable ? JanusGraphOperationStatus.OK : janusGraphGenericDao.lockElement(componentId, nodeType);
-        } catch (Exception e) {
-            lockElementStatus = JanusGraphOperationStatus.ALREADY_LOCKED;
-        }
-        return DaoStatusConverter.convertJanusGraphStatusToStorageStatus(lockElementStatus);
+        return lockComponentDefault(String componentId, NodeTypeEnum nodeType);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;componentId&#34;, delete this token